doc/Design.md

```diff
@@ -21,49 +21,65 @@ been backed up at some point in time. The state here means the content and meta
 data like the name and modification time for the file or the directory and its
 contents.
 
+*Storage ID*: A storage ID is the hash of the content of a file stored in the
+repository. This ID is needed in order to load the file from the repository.
+The storage hash is the SHA-256 hash of the content.
```

Comment:
I'm confused now... "Storage ID is the hash of the content" and "storage hash is the hash of the content". What's the difference between "storage ID" and "storage hash"? If it's the same thing, why mention both?